src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -696,7 +696,7 @@ protected virtual void ExecuteEvmCall(
             WorldState.Restore(snapshot);
 
         Complete:
-            if (!opts.HasFlag(ExecutionOptions.NoValidation))
+            if (!opts.HasFlag(ExecutionOptions.SkipValidation))
```

Comment:
I was worried that we might modify the header, but we probably are not modifying it?

Reply:
Not inside a single tx run as we aren't doing block processing